### PR TITLE
Changed graph colours.

### PR DESCRIPTION
**Pull request by olliejth**
Changed from an angry red to a lovely purple.

### CODE REVIEW

**Comment by Leon-JS02** [comment]
Such a lovely mauve.

**Review by Leon-JS02 — APPROVED** [review]
Stylin'

File: dashboard/longterm_transform.py [hunk 1]
```diff
@@ -27,10 +27,9 @@ def get_times_watered(df: pd.DataFrame) -> alt.Chart:
 
     title = alt.TitleParams(
         'Average daily water count per plant', anchor='middle')
-    bar_chart = alt.Chart(watered_df, title=title).mark_bar().encode(
+    bar_chart = alt.Chart(watered_df, title=title).mark_bar(color='#a05799').encode(
```
Comment (mehrdadhalali) [inline dashboard/longterm_transform.py:30]: Pretty colours

**Review by mehrdadhalali — APPROVED** [review]
Mehrproved

**Pull request merged**